crates/bevy_render/src/mesh/primitives/dim2.rs

```diff
@@ -405,3 +405,62 @@ impl From<Capsule2dMeshBuilder> for Mesh {
         capsule.build()
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use bevy_math::primitives::RegularPolygon;
+
+    use crate::mesh::{Mesh, VertexAttributeValues};
+
+    /// Sin/cos and multiplication computations result in numbers like 0.4999999.
+    /// Round these to numbers we expect like 0.5.
+    fn fix_floats<const N: usize>(points: &mut [[f32; N]]) {
+        for point in points.iter_mut() {
+            for coord in point.iter_mut() {
+                let round = (*coord * 2.).round() / 2.;
+                if (*coord - round).abs() < 0.00001 {
+                    *coord = round;
+                }
+            }
+        }
+    }
+
+    #[test]
+    fn test_regular_polygon() {
```

Comment:
Document the purpose of this test please. I'm not sure I could fix this if I broke it.

Reply:
It's not my test, it was added by https://github.com/bevyengine/bevy/pull/11017. Personally, I find it to be a bit redundant and would remove it, but it was on the old `RegularPolygon` type so I kept it for now.

Reply:
"Document RegularPolygon by writing a test" and then proceeds to write no documentation for the test.

Fair enough.